[Diagram] Board 도메인 다이어그램 작성

vwx12345 opened the pull request:
### 💻 작업 내용
- 디렉토리 이동
- boardName 은 과한 VO 같아서 삭제

### ✨ 리뷰 포인트

### 📝 메모
- board 자체의 도메인 로직은 없는 것 같아 수정사항이 많이 없습니다.

### 🎯 관련 이슈
#5

vwx12345 committed:
docs(diagram): boardName 은 string 으로 두는 방식으로 변경

vwx12345 committed:
chore(diagram): auto-generate SVGs

codingbaraGo commented on document/puml/domain/boardcontext/boardAggregate/boardAggregate.puml:
```diff
@@ -2,26 +2,19 @@
-        --
-        +validate()
-     }
     class Creator <<Value Object>> {
         +Long : memberId
      }
      
```
Creator는 게시판 개설자를 말하는거죠?
최고관리자가 아니라?
전에 board event 다이어그램쪽에서 조건 검색에 관련 내용을 본것같은데 그때는 최고관리자를 기준으로 검색을 한다 적어주셨던 것 같아서요

vwx12345 replied:
creator는 당장은 필요없을 것 같습니다. 최고 관리자가 맞습니다.

Commit:
docs(diagram): creator 삭제

Commit:
docs(diagram): creator -> manager로 변경

# Conflicts:
#	document/puml/domain/boardcontext/boardAggregate/boardAggregate.puml
#	document/puml/domain/boundedContextMap.puml

vwx12345 committed:
Merge branch 'develop' into document/diagram-domain-board/#5

vwx12345 committed:
chore(diagram): auto-generate SVGs
